schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -176,6 +176,9 @@
     "extensions": {
       "type": "object",
       "properties": {
+        "name": {
+          "type": "string"
+        },
```

Comment:
we don't need to add "name" definition here as it can be inherited. Same for "location".